File: src/main/java/uk/gov/justice/digital/hmpps/oauth2server/auth/repository/OauthServiceRepository.java
package uk.gov.justice.digital.hmpps.oauth2server.auth.repository;

import org.springframework.data.repository.CrudRepository;
import uk.gov.justice.digital.hmpps.oauth2server.auth.model.Service;

import java.util.List;

public interface OauthServiceRepository extends CrudRepository<Service, String> {
    List<Service> findAllByEnabledTrueOrderByName();

    List<Service> findAllByOrderByName();
}

File: src/main/kotlin/uk/gov/justice/digital/hmpps/oauth2server/auth/repository/OauthServiceRepository.kt
package uk.gov.justice.digital.hmpps.oauth2server.auth.repository

import org.springframework.data.repository.CrudRepository
import uk.gov.justice.digital.hmpps.oauth2server.auth.model.Service

interface OauthServiceRepository : CrudRepository<Service, String> {
  fun findAllByEnabledTrueOrderByName(): List<Service>
  fun findAllByOrderByName(): List<Service>
}
